doc/userguide/rules/hassh-keywords.rst

```diff
@@ -0,0 +1,63 @@
+Hassh Keywords
+==============
+
+Suricata comes with a Hassh integration (https://github.com/salesforce/hassh). Hassh is used to fingerprint ssh clients and servers.
+
+Hassh must be enabled in the Suricata config file (set 'app-layer.protocols.ssh.hassh' to 'yes').
+
+hassh
+-----
+
+Match on hassh (md5 of of hassh algorithms of client).
+
+Example::
+  alert ssh any any -> any any (msg:"match hassh"; \
+      hassh; content:"ec7378c1a92f5a8dde7e8b7a1ddf33d1";\
+      sid:1000010;)
+      
+``hassh`` is a 'sticky buffer'.
```

Comment:
I wonder if a sticky buffer makes sense for some md5...

Reply:
I was checking ja3/ja3s (it's sticky buffer) use cases. 
https://suricata.readthedocs.io/en/suricata-4.1.0/rules/ja3-keywords.html
So, I decided to make hassh 'sticky buffer' also.

Reply:
ok